test/test-files/sample_daemon_output.ts

```diff
@@ -0,0 +1,562 @@
+/**
+ * @license
+ * Copyright 2018 Balena Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const sampleDaemonOutput = [
+    { status: 'Pulling from library/registry', id: '2' },
+    { status: 'Pulling fs layer',
+    progressDetail: {},
+    id: 'd6a5679aa3cf' },
+    { status: 'Pulling fs layer',
```

Comment:
The indentation is weird, has this file been formatted with prettier?

Reply:
It hadn't -- this repo is not setup with resin-lint and prettify (not a task for this pull request...)
But I have installed prettify globally and I've now run it on this file and on `test_build_stream.ts`.